README.md

```diff
@@ -1 +1,196 @@
+
+- [Github Issues](https://github.com/pythonindia/inpycon2021-tasks/issues?q=is%3Aopen+is%3Aissue+label%3Awg%2Fcfp)
+- Meeting:
+- Chair: Sukanya
+- Co-chair:
+- Shadow:
+- Members
+
+### Content and Design
+
+- [Github Issues](https://github.com/pythonindia/inpycon-blog/issues/)
+- Meeting:
+- Chair: Vipul Gupta
+- Co-chair:
+- Shadow:
+- Members
+  * Nikhil Maan
+  * Arvind
+
+### Dev Sprint/Workshops
+- [Github Issues](https://github.com/pythonindia/inpycon2021-tasks/issues?q=is%3Aopen+is%3Aissue+label%3Awg%2Fdevsprint)
+- Meeting:
+- Chair:
+- Co-chair:
+- Shadow:
+- Members
+
+### Mentorship/Editorial
+
+- [Github Issues](https://github.com/pythonindia/inpycon2021-tasks/issues?q=is%3Aopen+is%3Aissue+label%3Awg%2Fmentorship)
+- Meeting:
+- Chair:
+- Co-chair:
+- Shadow:
+- Members
+
+### Platform
+
+- [Github Issues](https://github.com/pythonindia/inpycon2021-tasks/issues?q=is%3Aopen+is%3Aissue+label%3Awg%2Fplatform)
+- Meeting:
+- Chair:
+- Co-chair:
+- Shadow:
+- Members
+
+### Review
+
+- [Github Issues](https://github.com/pythonindia/inpycon2021-tasks/issues?q=is%3Aopen+is%3Aissue+label%3Awg%2Freview)
+- Meeting:
+- Chair:
+- Co-chair:
+- Shadow:
+- Members
+
+
+### Bofs
+
+- [Github Issues]()
+- Meeting:
+- Chair:
+- Co-chair:
+- Shadow:
+- Members
+
+### Outreach and Communication (social media, community partners)
+
+- [Github Issues](https://github.com/pythonindia/inpycon2021-tasks/issues?q=is%3Aopen+is%3Aissue+label%3Awg%2Fsocial-media)
+- Meeting:
+- Chair: Abhishek
+- Co-chair:
+- Shadow:
+- Members
+
+### Technology
+
+- [Github Issues](https://github.com/pythonindia/inpycon2021-tasks/issues?q=is%3Aopen+is%3Aissue+label%3Awg%2Ftechnology)
+- Meeting:
+- Chair: Nabarun
+- Co-chair:
+- Shadow:
+- Members
+
+### Finance
+
+- [Github Issues](https://github.com/pythonindia/inpycon2021-tasks/issues?q=is%3Aopen+is%3Aissue+label%3Awg%2Fbilling)
+- Meeting:
+- Chair:
+- Co-chair:
+- Shadow:
+- Members
+
+### Diversity
+
+- [Github Issues](https://github.com/pythonindia/inpycon2021-tasks/issues?q=is%3Aopen+is%3Aissue+label%3Awg%2Fdiversity)
+- Meeting:
+- Chair: Sukanya
+- Co-chair:
+- Shadow:
+- Members
+
+### Tickets
+
+- [Github Issues](https://github.com/pythonindia/inpycon2021-tasks/issues?q=is%3Aopen+is%3Aissue+label%3Awg%2Ftickets)
+- Meeting:
+- Chair:
+- Co-chair:
+- Shadow:
+- Members
+
+### R&D (brainstorm new ideas like: poster presentation, digital swags)
+
+- [Github Issues]
+- Meeting:
+- Chair: Sayan Chowdhury
+- Co-chair:
+- Shadow:
+- Members
+
+## Committees
+
+### Code of Conduct
+
+- [Github Issues](https://github.com/pythonindia/inpycon2021-tasks/issues?q=is%3Aopen+is%3Aissue+label%3Awg%2Fcode-of-conduct)
+- Meeting:
+- Chair:
+- Co-chair:
+- Shadow:
+- Members
+
+## Timeline
+### TLDR;
+* PyCon India 2021 formal proposal going live on 19th Feb, 2021.
+* Website going live on 01st Mar, 2021.
+* Ticket sales going live on 01st May, 2021.
+* CFP opens on March 01st and Closes on last week of May, 2021.
```

Comment:
Also, try this mid-May. Extend if needed. Better to have a calendar gives a more better idea. We can setup a Google public calendar like we did last year.